### ISSUE TITLE
Allow importing files with .graphql

### ISSUE THREAD

**Issue by josemarluedke** [csantero/broccoli-graphql-filter#2]
It would be very nice if this project generated modules that are compatible to importing with the suffix `.graphql`. 

For these using TS, it is easier to make TS happy if we import it with an extension. 

So, let's say we have a file called `account.graphql`. 

Today, we would import it using:

```js
import accountQuery from './account';
```

What I would like to have is: 

```js
import accountQuery from './account.graphql';
```

Same goes for importing fragments inside of graphql files.

**Comment by csantero** [comment]
I don't have any objection to this as long as it can be opted-into via configuration.

**Comment by josemarluedke** [comment]
This is now implemented with the option `keepExtension` released in v0.3.2
